MkDocs/hpu_8.6/index.md

```diff
@@ -1,89 +1,12 @@
-![img](/img/hpu_8.6/banner.png){:style="height:150px;width:800px"}
+![img](/img/hpu_8.6/banner.png)
```

Comment:
You reference these image files banner.png  and others but don't include them in your pull request.  You are required to make sure that your pull request is functionally complete without files missing or having errors.  Please add the missing image files to this PR.   I would also suggest not having 2 periods in a file name.  It is difficult to read and may cause problems with file type rendering for an image.

Reply:
All of the image files are showing up for me in the file I created.